helpers/pdfUtils.py

```diff
@@ -10,11 +10,10 @@
 
 def get_pdf_for_url(url):
   try:
-    tmp = tempfile.NamedTemporaryFile(delete=True)
-    child = subprocess.Popen(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)], stdout=subprocess.PIPE)
-    child.communicate()
-    contents = tmp.read()
-    tmp.close()
-    return contents
+    with tempfile.NamedTemporaryFile(delete=True) as tmp:
+      child = subprocess.call(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)])
+      contents = tmp.read()
+      tmp.close()
+      return contents
   except CalledProcessError, e:
     logger.exception("getting pdf failed for url %s: %s" % (url, e.output))
```

Comment:
Have you manually tested that the relevant error info gets logged?

Reply:
Yep! it does in fact get log

Reply:
CalledProcessError contains the output of the failed subprocess